### ISSUE TITLE
[CAL-2797] Troubleshooter v2 - Pull the calendar info for "busy" events

### ISSUE THREAD

**Issue by ciaranha** [calcom/cal.com#12752]
The v1 shipped with just showing busy for events

![CleanShot 2023-12-12 at 17.51.28@2x.png](https://uploads.linear.app/e86bf957-d82f-465e-b205-135559f4b623/12f16adf-18f4-4c7a-a681-f9e559816ded/75e40ada-2653-4132-944c-133f22132935?signature=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJwYXRoIjoiL2U4NmJmOTU3LWQ4MmYtNDY1ZS1iMjA1LTEzNTU1OWY0YjYyMy8xMmYxNmFkZi0xOGY0LTRjN2EtYTY4MS1mOWU1NTk4MTZkZWQvNzVlNDBhZGEtMjY1My00MTMyLTk0NGMtMTMzZjIyMTMyOTM1IiwiaWF0IjoxNzAyNDAzNjMzLCJleHAiOjE3MDI0OTAwMzN9.d-dS0AHBMx1Xy33DhIjXqRCnJ34Xd0JFgV3gd55wwNw)

We should show which calendar the event is coming from [as per designs](https://www.figma.com/file/UajFu4M1APhn2AywAEJkJr/New-Features?node-id=5966%3A558458&mode=dev).

![CleanShot 2023-12-12 at 17.53.13@2x.png](https://uploads.linear.app/e86bf957-d82f-465e-b205-135559f4b623/0f471220-7b7e-4d5e-9d43-319fd42702ea/dbf8a37c-12b7-4e21-b3b2-66410c820208?signature=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJwYXRoIjoiL2U4NmJmOTU3LWQ4MmYtNDY1ZS1iMjA1LTEzNTU1OWY0YjYyMy8wZjQ3MTIyMC03YjdlLTRkNWUtOWQ0My0zMTlmZDQyNzAyZWEvZGJmOGEzN2MtMTJiNy00ZTIxLWIzYjItNjY0MTBjODIwMjA4IiwiaWF0IjoxNzAyNDAzNjMzLCJleHAiOjE3MDI0OTAwMzN9.nghFIfz34xDE7BPVoFl27Em7JwBRwQDwKTGfvXS9540)

<sub>From [SyncLinear.com](https://synclinear.com) | [CAL-2797](https://linear.app/calcom/issue/CAL-2797/troubleshooter-v2-pull-the-calendar-info-for-busy-events)</sub>

**Comment by ciaranha** [comment]
Also requested here https://github.com/calcom/cal.com/issues/12749

**Comment by aspiers** [comment]
The same problem applies to the "Overlay my calendar" feature - is that caused by the same underlying limitation, or a separate issue?

**Comment by sean-brydon** [comment]
> The same problem applies to the "Overlay my calendar" feature - is that caused by the same underlying limitation, or a separate issue?

Yeah this is the same issue - in short - the getBusyTimes doesnt pull in any calendar titles as it is mainly called in a public endpoint. We will need to use getServerSide session and make changes to ALL calendar services to pull this info in. 

It was too much for V1 as it requires a lot of core code changes that were a bit out of scope

**Comment by keithwillcode** [comment]
Next step is to look and see if Google scope change needed to retrieve this data or if it already comes back. If not, consider if this is important enough to re-request scopes from users.
